src/task.ts

```diff
@@ -166,9 +166,12 @@ export abstract class CompoundTask extends DFTask {
         children.map((c) => (c.parent = this));
         this.firstError = undefined;
 
-        // If the task has no children, then it's completed by definition.
+        // If the task has no children, throw an error
```

Comment:
Can you link to this issue in the comment in the code:
`// https://github.com/Azure/azure-functions-durable-js/issues/424`

Otherwise it doesn't feel entirely clear to me _why_ this isn't allowed

Reply:
Done